operations/research-and-development/product/release-process/release-overview.md

```diff
@@ -31,6 +33,67 @@ Mattermost numbers stable releases in the following format:
 - Release frequency: As required
 - Example: v1.2.5, v1.2.6
 
+## Overview of Release Cycles
+
+Currently Mattermost Cloud releases occur on a 2-week cycle, but the goal is to release to frequently. Cloud and Self-Managed PRs don’t need to be cherry-picked to releases, except for any last minute bug fixes that get merged after the release branch is cut, and any necessary hotfixes.
+
+Schedule for Mattermost Cloud releases:
+ - (T-7): 1 week prior to the release day, master is fast-forwarded to the “cloud” branch.
```

Comment:
Is this another case of changing "fast-forward" to merge? If so: "...the `master` branch is merged to the Cloud branch."